content/applications/websites/ecommerce/products.rst

```diff
@@ -4,275 +4,402 @@
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
+- :doc:`Warehouse <../../inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses>`;
+- :guilabel:`Out-of-Stock`: enable :guilabel:`Continue Selling` to allow customers to place orders
+  even when the product is **out of stock**. Leave the box unchecked to **prevent orders**;
+- :guilabel:`Show Available Qty`: displays the available quantity left under a specified threshold
+  on the product page. The available quantity is calculated based on the :guilabel:`On hand`
   quantity minus the quantity already reserved for outgoing transfers.
 
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
-
-Selling as kit
---------------
-
-If you are selling non-prepackaged kits (i.e., the kits are made of individual products), we
-recommend you read the related documentation to keep track of your stock.
-
-.. seealso::
-   :doc:`../../inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping`
+.. _ecommerce-products-product-comparison:
 
 Product comparison
 ==================
 
-You can enable a **product comparison tool** for your eCommerce by going to
-:menuselection:`Website --> Configuration --> Settings --> Shop - Products`, and ticking
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
+To allow website visitors to compare products based on their attributes, go to
+:menuselection:`Website --> Configuration --> Settings`, scroll down to the
+:guilabel:`Shop - Products` section, and enable :guilabel:`Product Comparison Tool`.
 
-On the product page, scroll down to the :guilabel:`Specifications` section and click
-:guilabel:`Compare`. Repeat the same process for all products you wish to compare. Then, click the
-:guilabel:`Compare` button of the pop-up window at the bottom of the page to reach the comparison
-summary.
-
-.. note::
-   The :guilabel:`Product Comparison Tool` can only be used if :doc:`attributes <products/variants>`
-   are set on the **product's template**.
+The :icon:`fa-exchange` (:guilabel:`Compare`) icon is now available on each product card on your main
+shop page. Click the :icon:`fa-exchange` (:guilabel:`Compare`) option on the products you want to
+compare. Then, click :icon:`fa-exchange` :guilabel:`Compare` in the pop-up window at the bottom of
+the page to reach the comparison summary.
 
 .. image:: products/products-compare.png
-   :align: center
    :alt: Product comparison window
 
+.. note::
+   - The :guilabel:`Product Comparison Tool` is only available for products with
+     :ref:`attributes <ecommerce-products-product-variants>`.
+   - The :icon:`fa-exchange` (:guilabel:`Compare`) option can be specifically
+     :ref:`added or removed <ecommerce-products-customization>` on each product page.
```

Comment:
I'm not sure about this; it seems like the Compare option on the product page is global so you can't decide for each individual product if you want to have it or not (i.e., if you remove it, you remove it for *all product pages*).

I would suggest changing this entirely to instead mention the fact that you can you can also click Compare on a product page (like I mentioned in my first review).